temporal-test-server/src/main/java/io/temporal/internal/testservice/StateMachines.java

```diff
@@ -341,15 +340,17 @@ public int getAttempt() {
   }
 
   static final class NexusOperationData {
+    // Timeout for an individual Start or Cancel Operation request.
+    final java.time.Duration requestTimeout = java.time.Duration.ofSeconds(5);
```

Comment:
Yeah I mentioned in another comment that this is mostly just so the tests will run faster since there is no way to configure it since we don't have dynamic configs. I will switch it back to 10s and just increase the test timeout to make it consistent.

Reply:
If the test time is proving to be a big problem we can add some concept of dynamic config to the test server

Reply:
It's a little tricky to work with because when writing tests for timeouts, I have to work around the fact that long polls have a timeout of 10s and now so do the tasks, so I had to use sleeps to avoid a race condition where the poll would timeout first and fail the test.

If we are adding more such tests in the future, having some notion of dynamic config would be handy to keep the overall run time of the functional tests down, but I think it's okay for now.